server/cluster/table.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package cluster
+
+import "github.com/CeresDB/ceresdbproto/pkg/clusterpb"
+
+type Table struct {
+	schema *clusterpb.Schema
+
+	meta *clusterpb.Table
+}
+
+func (t *Table) getID() uint64 {
```

Comment:
Should it be exposed for use:
```suggestion
func (t *Table) GetID() uint64 {
```